### PR TITLE
Improves the quality of life of Cluwnes

### PR DESCRIPTION
**Pull request by Fox-McCloud**
By ensuring they cannot escape their fate.

HUNKKKKKKEEEEE!

- A cluwne is now properly renamed; their original name cannot be restored
- A cluwne's honk tumor is now impossible to remove
- A cluwne's brain damage and fatness is re-applied every so often
- Cluwne mask can no longer be melted with acid
- Cluwne suit no longer has sensors
- Removed the animation from cluwning because it causes some problems with directionality of sprites
- If you rip out a Cluwne's brain and put it in a new body, it's going to become a cluwne, too
- Cloning a Cluwne will end up making them a Cluwne again, post cloning process
- Cannot remove Cluwne mutations with mutadone

*There* **is no** ***ESCAPE***

:cl: Fox McCloud
tweak: Cluwning is far more difficult to remove
tweak: Cluwne mask cannot be melted with acid
tweak: Cluwne suit has no sensors
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine.dm [hunk 1]
```diff
@@ -686,6 +686,9 @@
 	color = "#5096C8"
 
 /datum/reagent/medicine/mutadone/on_mob_life(mob/living/carbon/human/M)
+	if(M.mind && M.mind.assigned_role == "Cluwne") // HUNKE
```
Comment (Fox-McCloud) [inline code/modules/reagents/chemistry/reagents/medicine.dm]: This is disgusting and snowflakey, but the alternative was to delete mutadone by the honk tumor every `on_life` *and* re-add the mutations that it removed, which is expensive; it's better, performance and cleanliness wise, to just have this here.

Comment (Anticept) [inline code/modules/reagents/chemistry/reagents/medicine.dm]: They are a snowflakey super special case, so all things considered, it's fine.

File: code/datums/spells/cluwne.dm [hunk 2]
```diff
@@ -13,7 +13,11 @@
 
 /mob/living/carbon/human/proc/makeCluwne()
 	to_chat(src, "<span class='danger'>You feel funny.</span>")
-	adjustBrainLoss(80)
+	if(!get_int_organ(/obj/item/organ/internal/brain/cluwne))
+		var/obj/item/organ/internal/brain/cluwne/idiot_brain = new
+		idiot_brain.insert(src, make_cluwne = 0)
+		idiot_brain.dna = dna.Clone()
```
Comment (Crazylemon64) [inline code/datums/spells/cluwne.dm]: Are you sure this won't cause weirdness with having more than one brain, or making anticluwning no longer work?

**Comment by GunDOSMk1** [comment]
WHUT nuuuuuuu, don't du this! I'll have to delete my guide to fixin them! *sobs*

File: code/datums/spells/cluwne.dm (hunk 2 above)
Comment (Fox-McCloud) [inline code/datums/spells/cluwne.dm]: Yeah, I specifically tested for this; it overrides+replaces the existing brain.

AntiCluwning would still work.

**Comment by Ssirius** [comment]
@GunDOSMk1 No need, there is still a clear and easy way to fix Cluwnes, you kill the Cluwn then gib/cremate/destroy the body, and remember the person for what they once were, and not that witch they became.

**Comment by ktoma36** [comment]
Does this affect cyborgs? It would be really strange if a cyborg were to suddenly burst into a cluwn

File: code/datums/spells/cluwne.dm (hunk 2 above)
Comment (Crazylemon64) [inline code/datums/spells/cluwne.dm]: What if you brain-transplanted an AntiCluwne?

**Comment by Spartan6** [comment]
You need cyborgs to detonate if a cluwne brain enters and they become a bog standard cluwne again.

**Comment by MarcellusPye** [comment]
My one problem with this is that the wizard spell is essentially round over now.

File: code/datums/spells/cluwne.dm (hunk 2 above)
Comment (Fox-McCloud) [inline code/datums/spells/cluwne.dm]: He's still a cluwne in soul~

**Comment by Bigtimetripod** [comment]
@marcelluspye

Ever heard of "EL'NATH!" being yelled by wiznerd? Now thats a round ender, not some damn cluwne spell.

**Comment by matt81093** [comment]
What happens to CLUWNES whose brains are implanted into borgs?

**Comment by taukausanake** [comment]
And for that matter what about cluwn brains that are put into spider bots? I'm all for making the poor soul suffer forever but there has to be some form of holding the brain without cluwnism.

Actually, I vote for a cluwn borg. Whenever a cluwn brain is put into a borg it gets overrode by a cluwn module. Still trapped by the curse but at least then it could be a little more useful to the station

**Comment by Rumiluntti** [comment]
>My one problem with this is that the wizard spell is essentially round over now.

That's the point. You could also be dead, just as easy.

>Still trapped by the curse but at least then it could be a little more useful to the station

You seem to miss the point of cluwnes. They need to suffer until the sweet release of death, not be useful

**Comment by taukausanake** [comment]
Fine then, just make a cluwn bot that's just suffering and pitiful to look at. Also they could be useful more as a borg than as a brain. Then you have something to beat and it won't completely die still. The borg frame will have to be destroyed though.

Do you think it wouldn't be such a bad idea to make a cluwn module and the spell to work on borgs too? Maybe something similar with the horse curse, have mountable borgs and use them as cattle

**Comment by Fox-McCloud** [comment]
>My one problem with this is that the wizard spell is essentially round over now.

Keep in mind, that they could have gone for Ei Nath, over this; they both have the same cost, cooldown, and cooldown decrease per level. Cluwne makes someone a potential kill target by the crew, but leaves them able to still attack/defend themselves/try to kill the wizard.

Ei Nath, on wizard rounds, is mostly just a control-alt-delete of the mob it's used on.

**Comment by johannhawk** [comment]
For those that do not remember, in the land where cluwning originates (goon), you can escape cluwning with borging, listening to honkhonkbutt is also an acceptable source to back up this statement.

**Comment by Rumiluntti** [comment]
Upgraded cluwning

**Comment by matt81093** [comment]
I wouldn't mind seeing CLUWNE borgs at some point.. not sure about them in spiderbots though.. cluwnebots.. misserably honking around dropping bannanna's everywhere?

**Pull request merged**